src/common-utils/scripts/bash_theme_snippet.sh

```diff
@@ -18,7 +18,7 @@ __bash_prompt() {
         fi`'
```

Comment:
Can we bump the patch version in here, thanks! 
https://github.com/devcontainers/features/blob/c408d345524651c8c0224520e193454c18b88a81/src/common-utils/devcontainer-feature.json#L3

Reply:
I added the commit. Please let me know if something is wrong.